### PR TITLE
Document the `params IDiagnosticsSubscriber[] subscribers` parameter.

### PR DESCRIPTION
**Pull request by gregkalapos**
Addressing #680. 

Short description on how you can manually turn on tracing components.

### CODE REVIEW

File: docs/setup.asciidoc [hunk 1]
```diff
@@ -64,6 +64,17 @@ With this you enable every agent component including ASP.NET Core tracing, monit
...
 
+In case you would like to turn on specific tracing components you can pass those to the `UseElasticApm` method.
+
+For example:
+
+[source,csharp]
+----
+app.UseElasticApm(Configuration,
+	new HttpDiagnosticsSubscriber(),  /* Enable tracing of outgoing HTTP requests */,
+	new EfCoreDiagnosticsSubscriber() /* Enable tracing of database calls through EF Core*/)
```
Comment (bmorelli25) [inline docs/setup.asciidoc]: Is it standard to put the closing parenthesis after the inline comment? If so, LGTM.

File: docs/setup.asciidoc [hunk 2]
```diff
@@ -64,6 +64,17 @@ With this you enable every agent component including ASP.NET Core tracing, monit
...
 In case you only reference the `Elastic.Apm.AspNetCore` package, you won't find the `UseAllElasticApm`. Instead you need to use the `UseElasticApm()` method from the `Elastic.Apm.AspNetCore` namespace. This method turns on ASP.NET Core tracing, and gives you the opportunity to manually turn on other components. By default it will only trace ASP.NET Core requests - No HTTP request tracing, database call tracing or any other tracing component will be turned on.
 
+In case you would like to turn on specific tracing components you can pass those to the `UseElasticApm` method.
+
+For example:
+
+[source,csharp]
+----
+app.UseElasticApm(Configuration,
+	new HttpDiagnosticsSubscriber(),  /* Enable tracing of outgoing HTTP requests */,
```
Comment (bmorelli25) [inline docs/setup.asciidoc]:
```suggestion
	new HttpDiagnosticsSubscriber(),  /* Enable tracing of outgoing HTTP requests */
```

Comment (bmorelli25) [reply docs/setup.asciidoc]: Sorry, just noticed an extra comma here

File: docs/setup.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply docs/setup.asciidoc]: Looking at how this is rendered with a scrollbar in our docs, I'd be tempted to move that parenthesis to before the comment:

Screen Shot 2020-01-14 at 11 13 31 AM

Comment (gregkalapos) [reply docs/setup.asciidoc]: > Is it standard to put the closing parenthesis after the inline comment?

I pasted it to my editor and auto-formatted, but as it seems the C# auto-formatter with our configs does not care about it, it leaves both versions as it is. I took your suggestion, let's roll with that.

**Pull request merged**